### PR TITLE
Added backups playbook and role

### PR DESCRIPTION
**Pull request by ericm**

### CODE REVIEW

File: setup-backups.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- name: "Ensure backup job on each proxmox host"
+  hosts: proxmox_hosts
+  roles: backups
+  vars:
+    backup_day: "6"
+    backup_hour: "0"
+    router_hostname: "router.vm.netsoc.co"
```
Comment (ocanty) [inline setup-backups.yml:7]: make this `ignore: ["router.vm.netsoc.co"]`

Comment (ocanty) [reply setup-backups.yml:7]: add exclude_paths: [list]

**Comment by ocanty** [comment]
https://forum.proxmox.com/threads/vzdump-only-disk-1.50136/

**Comment by ocanty** [comment]
think we could do daily backups? if we can schedule them at a certain time (~3am) we can minimize performance loss

**Comment by ocanty** [comment]
also does the webhook get called for each server or just as a whole, servers like feynman have shit i/o and are slow so Id like to know the difference in backup speed

**Comment by ericm** [comment]
Webhook callled on each since cron is host specific
The name is set to the hostname so itll look like each server is posting in sysadmin alerts

**Comment by ocanty** [comment]
> Webhook callled on each since cron is host specific
> The name is set to the hostname so itll look like each server is posting in sysadmin alerts

haven't looked at the source yet but could you genericify the webhook url so can use it for other things

**Comment by ocanty** [comment]
also pls document when merged https://wiki.netsoc.co/en/sysadmin/backups

**Comment by ericm** [comment]
If you setup duplicity locally and get the refresh token, just add it to secrets

**Pull request closed without merging**